lib/msf/core/post/file.rb

```diff
@@ -582,6 +582,29 @@ def copy_file(src_file, dst_file)
   end
   alias :cp_file :copy_file
 
+  #
+  # Return a list of the Windows Drives
+  #
+  def get_drives
+    if session.type != "meterpreter" && session.platform != 'windows'
```

Comment:
Can we make this work for shell and powershell sessions as well? We've been trying to move the Post API towards compatibility across the different session types. Should also note that this should be compatible with the Python meterpreter which has railgun support like Windows does.